addStyles.js

```diff
@@ -171,12 +171,17 @@ function attachTagAttrs(element, attrs) {
 function addStyle(obj, options) {
 	var styleElement, update, remove;
 
+	if (obj.sourceMap && typeof options.sourceRoot === "string") {
```

Comment:
Yeah, I would rather go with this than to add yet another field to the API. Better reuse this time.